toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -572,6 +572,7 @@ func (im *IsoMaker) copyAndRenameConfigFiles() (err error) {
 // files can be used by custom initrd/LiveOS images that will look for them
 // on the iso media.
 func (im *IsoMaker) copyIsoAdditionalFiles() (err error) {
+	logger.Log.Debugf("Copying ISO additional files.")
```

Comment:
nit: please remove `.` at the end, as part of the cleanup effort for logging

Reply:
Fixed